app/services/moab_on_storage/storage_directory.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'find'
+
```

Comment:
add comment:  "distinguish between the moab record in the db and the Moab on local storage (and the zipped moab versions replicated to the cloud)"  ?